csharp/dotnet-windows/console/Samples/Samples/Program.cs

```diff
@@ -209,5 +214,70 @@ public static async Task GetContentResult()
                 connection.Close();
             }
         }
+
+        // Pronunciation assessment with enableProsodyAssessment
+        // See more information at https://aka.ms/csspeech/pa
+        public static async Task GetProsodyResult()
+        {
+            // Creates an instance of a speech config with specified subscription key and service region.
+            // Replace with your own subscription key and service region (e.g., "westus").
+            var config = SpeechConfig.FromSubscription("YourSubscriptionKey", "YourServiceRegion");
+            
+            //set to true to get the prosody value
+            bool enableProsodyAssessment = true;
```

Comment:
For `GetProsodyResult`, `enableProsodyAssessment` should always be `true`. So you can use hard code in line 241.

```C#
 string jsonConfig = "{\"GradingSystem\":\"HundredMark\",\"Granularity\":\"Phoneme\",\"EnableMiscue\":\"true\", \"EnableProsodyAssessment\":\"true\"}";
```

Reply:
fixed and removed the variable